thefuck/rules/remove_shell_prompt_literal.py

```diff
@@ -1,4 +1,4 @@
-"""Fixes error for commands containing the shell prompt symbol '$'.
+"""Fixes error for commands containing 1 or more occurrences of the shell prompt symbol '$'.
```

Comment:
Nit: try to keep it under 80 cols

```suggestion
"""Fixes error for commands containing one or more occurrences of the shell
prompt symbol '$'.
```